### DISCUSSION TITLE
Why don't use and Imports statements inside Webpack?

### DISCUSSION THREAD

**Discussion by ghost** [webpack/webpack#15861] [General]
Hi, why do I always get the `Cannot use imports statement` error when using Webpack? 
It’s not for the` babel-presets` because I've them installed and configured...

![Captura de pantalla 2022-05-25 a las 0 46 01](https://user-images.githubusercontent.com/22520811/170145827-f1a8eb32-1677-4607-9858-bb06e50b8977.png)

**Comment by alexander-akait** [comment]
Do you have `type: "module"` in `package.json`?

**Reply by ghost** [reply]
Yes, but if I remove it, I get another error and it doesn’t let me compile or work with Webpack. At the end I send Webpack to the trash and work with `Vite.js`

**Reply by alexander-akait** [reply]
@sergiojurcas You use `require` in module package, what you expected? There is no problem with webpack, just rename `webpack.config.js` to `webpack.config.cjs` and please read how ECMA modules work in Node.js (and not only)

**Reply by ghost** [reply]
Yes, I know that but I've tried in the past to work with extension `. js` and not `. cjs o . mjs` and it worked, I've a GIST with configuration files and in the past they worked for me.

**Reply by alexander-akait** [reply]
Most likely you updated Node.js and now your Node.js version supports `type: "module"`

**Reply by ghost** [reply]
Yep, I've got node version 16.15 but I did what you said and it still doesn’t work...

![Captura de pantalla 2022-05-25 a las 16 08 16](https://user-images.githubusercontent.com/22520811/170282580-b5a62136-3018-463e-98e8-6ad381a1aa85.png)

**Reply by alexander-akait** [reply]
Looks like you have other files in ESM, in this case, make sense to use ESM modules everywhere, go to `webpack.config.js` and change `require` on `import` and `module.exports` on `export default`
